src/Moryx.AbstractionLayer/Drivers/Camera/ICameraDriver.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Moryx.AbstractionLayer.Drivers;
```

Comment:
I wonder how this makes it into this PR, that's already merged 🤔

Reply:
We found this to happen from time to time, Github just recognized changes from older merges that actually don't change anything🙈